.github/CODEOWNERS

```diff
@@ -1393,6 +1398,8 @@ x-pack/test/api_integration/deployment_agnostic/services/ @elastic/appex-qa
 x-pack/test/**/deployment_agnostic/ @elastic/appex-qa #temporarily to monitor tests migration
 
 # Core
+/x-pack/test/banners_functional @elastic/kibana-core
```

Comment:
`appex-sharedux` own the banner feature. This should move to the # Shared UX section

Reply:
Yes ma'am!  On it!